its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/SQLServerTest.java

```diff
@@ -63,7 +63,7 @@ void should_find_issues_in_cs_files() throws Exception {
     List<Issue> issues = TestUtils.allIssues(ORCHESTRATOR);
     assertThat(issues).hasSize(4);
     assertThat(TestUtils.getMeasureAsInteger(getFileKey(), "ncloc", ORCHESTRATOR)).isEqualTo(19);
-    assertThat(TestUtils.getMeasureAsInteger(PROJECT_KEY, "ncloc", ORCHESTRATOR)).isEqualTo(25);
+    assertThat(TestUtils.getMeasureAsInteger(PROJECT_KEY, "ncloc", ORCHESTRATOR)).isEqualTo(36);
```

Comment:
There is an auto-generated file in obj, as follows:

![image](https://github.com/user-attachments/assets/497f6f7b-e08b-49de-bf44-c4a53e129a89)

I think we want to exclude the `obj` and the `bin` directories, right?
I will make another ticket if that is the case.

Reply:
We discussed internally and decided to ignore the `bin/` and `obj/` directories.